java/zulu-jdk/Portfile

```diff
@@ -46,6 +44,26 @@ subport zulu-jdk7 {
     default_variants           +without_xawt
 }
 
+subport zulu-jdk8 {
+    name                       zulu-jdk8
```

Comment:
Don't change name here, it should always match the directory. If you need the name of the subport somewhere, use $subport.

Reply:
@jmroot good catch! Thanks, it was a copy-and-paste issue.